src/run/next.cts

```diff
@@ -97,11 +92,9 @@ export async function getMockedRequestHandler(...args: Parameters<typeof getRequ
       } catch (error) {
         // only try to get .html files from the blob store
         if (typeof path === 'string' && path.endsWith('.html')) {
-          const store = getRegionalBlobStore()
+          const cacheStore = getMemoizedKeyValueStoreBackedByRegionalBlobStore()
           const relPath = relative(resolve('.next/server/pages'), path)
-          const file = (await store.get(await encodeBlobKey(relPath), {
-            type: 'json',
-          })) as HtmlBlob | null
+          const file = await cacheStore.get<HtmlBlob>(relPath, 'staticHtml.get')
```

Comment:
It's probably automatic, but I'm just calling it out in case - I see `blobStore.set` and `tagManifest.set` but not seeing `staticHtml.set` anywhere.

Reply:
We upload those only at build time ( https://github.com/opennextjs/opennextjs-netlify/blob/b5754f40721dc8438bcb1aaf1df1a2727a726a36/src/build/content/static.ts#L42-L46 which will make use of [File-based blobs uploads](https://docs.netlify.com/blobs/overview/#file-based-uploads). As we only copy and let `@nelify/cli` or `buildbot` upload those blobs after build command - we don't create build-time traces for that.

We don't add/update those anymore once deployed